tests/NHSD.BuyingCatalogue.Ordering.Services.UnitTests/IdentityUserTests.cs

```diff
@@ -2,10 +2,10 @@
 using System.Diagnostics.CodeAnalysis;
 using AutoFixture.NUnit3;
 using FluentAssertions;
-using NHSD.BuyingCatalogue.Ordering.Application.Services;
+using NHSD.BuyingCatalogue.Ordering.Services;
```

Comment:
Is this `using` statement required?

Reply:
Removed `using `statements ` NHSD.BuyingCatalogue.Ordering.Services`